script/check-posix-pty

```diff
@@ -139,7 +139,7 @@ EOF
 
     netstat -an | grep -q 61631 || die 'TMF port is not available!'
 
-    extaddr=$(awk '/extaddr/{getline; print}' $OT_OUTPUT | tr -d '\r\n')
+    extaddr=$(grep -aoE '[0-9a-z]{16}' $OT_OUTPUT)
```

Comment:
I think we need to find (and fix) the root cause. 
FYI: I tested it locally on my Ubuntu 18.04 PC and it didn't fail.